pkg/secrets/secrets_test.go

```diff
@@ -0,0 +1,37 @@
+package secrets
+
+import (
+	"testing"
+
+	oconfig "github.com/openshift/api/config/v1"
+)
+
+func Test_processTags(t *testing.T) {
```

Comment:
TestProcessTags would follow the naming convention throughout the repo.

Reply:
Sure. I leaned towards the standard to identify a test for a non-exported function.